<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #252525;
color:#f9f9f9;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Pro Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите новое значение в ячейке</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCellContent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Pro Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#ffffff;
color:#000000;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveCellContent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Pro Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:green;
border-radius:4;
color:#252525;</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCanselSaveCellContent">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Pro Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:red;
border-radius:4;
color:#ffffff;</string>
   </property>
   <property name="text">
    <string>Cansel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
